typescript/test/test_application_ts/src/index.ts

```diff
@@ -103,3 +104,18 @@ async function testCollect(client: showcase.v1beta1.EchoClient) {
   const expectedresponse = {content: words.join(' ')};
   assert.deepStrictEqual(result, expectedresponse);
 }
+
+async function testPagedExpand(client: showcase.v1beta1.EchoClient) {
+  const words = ['nobody', 'ever', 'reads', 'test', 'input'];
+  const request = {
+    content: words.join(' '),
+    pageSize: 2,
+  };
+  const [response] = await client.pagedExpand(request);
+  const result = response as Array<JSON>;
+  const expectedResponse = [];
+  for(var i = 0; i < words.length; i++){
```

Comment:
`expectedResponse = words.map(word => { content: word });`

Reply:
This is a better way but tsc compile will complain about the unused label `content`

Reply:
But I can `"allowUnusedLabels": true,` to the tsconfig file if that's ok